<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkeletonSideWidget</class>
 <widget class="QWidget" name="SkeletonSideWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Available skeletons</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <widget class="GroupBoxExtension" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Skeletons</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="ListWidgetExtension" name="skeleton_listview">
        <property name="minimumSize">
         <size>
          <width>212</width>
          <height>80</height>
         </size>
        </property>
        <property name="viewMode">
         <enum>QListView::ListMode</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="BaseEntityNameEdit" name="addName" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionGhj">
   <property name="text">
    <string>ghj</string>
   </property>
  </action>
  <action name="actionGhjghjg">
   <property name="text">
    <string>ghjghjg</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupBoxExtension</class>
   <extends>QGroupBox</extends>
   <header>GUI/WidgetExtensions/GroupBoxExtension.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BaseEntityNameEdit</class>
   <extends>QWidget</extends>
   <header location="global">GUI/BasicWidgets/BaseEntityNameEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListWidgetExtension</class>
   <extends>QListWidget</extends>
   <header>GUI/WidgetExtensions/ListWidgetExtension.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
